collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -337,10 +337,14 @@ k3s-rancher() {
 
   if [ -d /var/lib/rancher/k3s/server ]; then
     unset KUBECONFIG
-    k3s kubectl get events --all-namespaces > $TMPDIR/k3s/kubectl/events 2>&1
-    k3s kubectl get endpoints -o wide --all-namespaces > $TMPDIR/k3s/kubectl/endpoints 2>&1
-    k3s kubectl get configmaps --all-namespaces > $TMPDIR/k3s/kubectl/configmaps 2>&1
-    k3s kubectl get namespaces > $TMPDIR/k3s/kubectl/namespaces 2>&1
+    K3S_OBJECTS=(clusterroles clustrerrolebindings crds mutatingwebhookconfigurations namespaces nodes pv validatingwebhookconfigurations)
+    K3S_OBJECTS_NAMESPACED=(apiservices configmaps deployments daemonsets endpoints events helmcharts hpa ingress jobs leases pods pvc replicasets roles rolebindings statefulsets)
+    for OBJECT in "${K3S_OBJECTS[@]}"; do
+      k3s kubectl get ${OBJECT} > $TMPDIR/k3s/kubectl/${OBJECT} 2>&1
```

Comment:
Any reason not to use `-o wide`? That will give a few more fields in the output that might be useful.